dotnetv3/S3/RestoreArchivedObjectExample/RestoreArchivedObject.cs

```diff
@@ -31,7 +32,7 @@ public static void Main()
         /// <summary>
         /// This method restores an archived object from an Amazon S3 bucket.
         /// </summary>
-        /// <param name="client">The initialized S3 client object used to call
+        /// <param name="client">The initialized Amazon S3 client object used to call
         /// RestoreObjectAsync.</param>
         /// <param name="bucketName">A string representing the name of the
         /// bucket where the object to restore was located.</param>
```

Comment:
was located. -> is located?

Reply:
Where the object was located before being archived.